cmd/utils/flags.go

```diff
@@ -376,6 +376,11 @@ var (
 		Usage: "Ethereum L1 contract address",
 		Value: "",
 	}
+	L1RollupAddressFlag = cli.StringFlag{
```

Comment:
This will conflict with the etrog changes, I wonder if we should consolidate them - seems neat to just pass the zk config into the l1 syncer rather than pieces (not a blocker for this PR though just a note)

Reply:
okay, let's keep that in mind and I will resolve this conversation to not block the merging